src/server/enrichment-map/enrichment/enrichment.js

```diff
@@ -13,6 +13,20 @@ const _ = require('lodash');
 const path = require("path");
 
 
+// remove #WARNING and #INFO
+const parseBody = (body) => {
```

Comment:
use better variable names, e.g:
 parseBody -> parseGProfilerResponse
 body -> gProfilerResponse

These are examples where is is helpful to be less generic